src/sonic-yang-models/tests/yang_model_tests/tests_config/acl.json

```diff
@@ -789,5 +789,36 @@
                 ]
             }
         }
+    },
+    "ACL_TABLE_CTRLPLANE_ACCEPT_ACTION": {
```

Comment:
The test name can be "ACL_PACKET_ACTION_VALIDATE_VALUE_ACCEPT", i.e. test name must be closer to describing what are we testing, instead of the config. Thx.